js/components/SpidLoginButton.js

```diff
@@ -239,6 +242,18 @@ class IdpSelectionScreen extends React.Component {
     })
   }
 
+  createErrorMessage = () => {
+    if (this.props.userState.isError === true) {
```

Comment:
this check should happen at the caller, this function is named `createErrorMessage` and not `createErrorMessageOnlyIfUserStateIsError`